projects/ui-components/internal-shared/strings/en.json

```diff
@@ -9,7 +9,7 @@
   "I_HAVE_FEEDBACK": "I have feedback",
   "I_HAVE_SUGGESTION": "I have a suggestion",
   "YOU_MAY_CONTACT_ME_ABOUT_FEEDBACK": "You may contact me about this feedback",
-  "IF_YOUD_LIKE_TO_JOIN_OUR_CUSTOMER_PANEL": "If you'd like to join our Customer Panel, ",
+  "IF_YOUD_LIKE_TO_JOIN_OUR_CUSTOMER_PANEL": "If you'd like to join our Customer Panel,",
```

Comment:
I see you remove the space here but it isn't removed in the html.
Is that because the link directly after it adds a space?

Reply:
Yes. The act of putting the link on a separate line in the html creates whitespace in the UI

Reply:
That seems fishy, because then the formatting could remove or add the space. The spaces in the angular html should be escaped so they don't appear in the html.